Samples/AudioPreprocessing/WinUI/AudioPreprocessing/AudioPreprocessing/AudioPreprocessing - Backup.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
is this a temp file that you don't actually want to check in? (I'm assuming from the " - backup" suffix)

Reply:
Removed!